### PR TITLE
Dev bell

### PR DESCRIPTION
**Pull request by Anelon**
All options select-able for the customer to make a reservation, also added code for a rest controller to look up flights using arrival and departure airport.

### CODE REVIEW

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -1,15 +1,174 @@
...
 <head>
-<meta charset="ISO-8859-1">
-<title>Hello</title>
+    <meta charset="ISO-8859-1">
+    <title>Hello</title>
 </head>
 <body>
 <img src="kanaAirlines.png" alt="Kana Airlines">
-<h2>Hello World!</h2>
+<h2>Book a flight!</h2>
```
Comment (espresso64) [inline src/main/resources/templates/index.html:9]: Nice menu implementation, the dynamic data returns accurate results. Tested to work locally.

File: src/main/java/cst438flights/controller/flightRestController.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package cst438flights.controller;
+
+import cst438flights.domain.Flight;
+import cst438flights.domain.FlightInfo;
```
Comment (espresso64) [inline src/main/java/cst438flights/controller/flightRestController.java]: rest controller functions well with the main menu

**Review by espresso64 — COMMENTED** [review]
Code looks good, menu system functions great.

File: src/main/java/cst438flights/controller/flightRestController.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        FlightInfo flightInfo = flightService.getFlightInfo(arrivalAirport);
+
+        if (flightInfo == null) {
+            return new ResponseEntity<FlightInfo>(HttpStatus.NOT_FOUND);
+        } else {
+            return new ResponseEntity<FlightInfo>(flightInfo, HttpStatus.OK);
+        }
+    }
+
+    @GetMapping("/api/flights/{departureAirport}/{arrivalAirport}")
```
Comment (Nhinds88) [inline src/main/java/cst438flights/controller/flightRestController.java:33]: This looks good, I was able to pull the json easily

File: src/main/java/cst438flights/controller/flightRestController.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+import java.util.List;
+
+@RestController
+public class flightRestController {
+
+    @Autowired
+    private FlightService flightService;
+
+    @GetMapping("/api/flights/{flight_ID}")
```
Comment (Nhinds88) [inline src/main/java/cst438flights/controller/flightRestController.java]: I think this is left over from my original code base, its not pulling the JSON properly. I'll have to look into this.

File: src/main/resources/templates/index.html [hunk 5]
```diff
@@ -1,15 +1,174 @@
...
+<form action="#" th:action="@{/customer}" method="post">
+    <h3>Look up previous flights</h3>
+    <input type="text" name="email" id="email" value="email">
+    <br />
+    <input type="password" name="password" id="password">
+    <br />
+    <input type="submit" value="submit">
+</form>
+
+<script>
```
Comment (Nhinds88) [inline src/main/resources/templates/index.html:68]: Your JS here really pulls this part together, great fix!

**Review by Nhinds88 — COMMENTED** [review]
Tested locally, the home page looks and works great with your updates and your API works well also.

**Pull request merged**